customers/tests/test_customers_mutations.py

```diff
@@ -38,48 +48,66 @@
 @pytest.mark.parametrize(
     "api_client", ["berth_services"], indirect=True,
 )
-@freeze_time("2020-01-01T08:00:00Z")
-def test_create_boat_certificate(api_client, boat):
-    boat_id = to_global_id(BoatNode, boat.id)
-    file_name = "certificate_file.pdf"
-    check_date = today()
+def test_create_boat(api_client, customer_profile, boat_type):
+    check_date = today().date()
+    file_name = "certificate.pdf"
+    owner_id = to_global_id(ProfileNode, customer_profile.id)
+    boat_type_id = str(boat_type.id)
 
     variables = {
-        "boatId": boat_id,
-        "file": SimpleUploadedFile(
-            name=file_name, content=None, content_type="application/pdf"
-        ),
-        "certificateType": random.choice(list(BoatCertificateType)).name,
-        "validUntil": check_date,
-        "checkedAt": check_date,
```

Comment:
how did it work like this before 🤯 🤔

Reply:
Huh? 🤔

Reply:
these are date objects. how did they equal to strings like `"2020-07-02"`? this is what we faced now when fixing the tests

Reply:
Aaahh, yeah. It wasn't working before, it was _another_ of the failures that you helped me fix yesterday.

Reply:
nono, it was working when it was a test for `createBoatCertificate` 🤔  which is why I am wondering, how did it work